scripts/reindex.py

```diff
@@ -104,9 +110,14 @@ def main(argv: List[str]):
                        create_indices=args.create or args.index and args.delete)
 
     if args.index:
+        slot_manager = None
         logger.info('Queuing notifications for reindexing ...')
         num_notifications = 0
         for catalog in args.catalogs:
+            if slot_manager is None and isinstance(azul.repository_plugin(catalog),
```

Comment:
We should add a command `--no-slots` command line flag with which the user can suppress the slot management completely.